oak/common/app_config_serializer.cc

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "oak/common/app_config.h"
+#include "oak/common/utils.h"
+
+#include <csignal>
+#include <memory>
+#include <string>
+
+#include "absl/flags/flag.h"
+#include "absl/flags/parse.h"
+#include "asylo/util/logging.h"
+
+ABSL_FLAG(std::string, module, "", "File containing the compiled WebAssembly module");
```

Comment:
This will not work for applications with multiple modules, right? Perhaps we should have a flag mapping names to modules, or something along those lines?

@daviddrysdale WDYT?

Reply:
A map would be ideal for multi-Node applications but isn't natively supported by `ABSL_FLAG`.  A flag of type `vector<string>` is [supported](https://github.com/abseil/abseil-cpp/blob/master/absl/flags/marshalling.h#L35) but then needs some clear & stable ordering for `WebAssemblyConfiguration` items (alpha on `name`?)

Reply:
If I understand correctly (after today's discussion with @tiziano88) at some point all applications will be compiled in a single module, and will be run it by specifying an entry point.